Author: Sven Joachim <svenjoac@gmx.de>
Description: Use linux2.2 in linux terminfo
 Using the linux2.6 or linux3.0 terminfo entries breaks the display of
 line drawing characters in non-UTF-8 locales.
Bug-Debian: http://bugs.debian.org/665959
Last-Update: 2012-04-01

---
 misc/terminfo.src |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -953,7 +953,7 @@
 
 # This is Linux console for ncurses.
 linux|linux console,
-	use=linux3.0,
+	use=linux2.2,
 
 # Subject: linux 2.6.26 vt back_color_erase
 # Changes to the Linux console driver broke bce model as reported in
